.github/workflows/publish-pypi.yml

```diff
@@ -0,0 +1,46 @@
+name: Publish to (Test)PyPI
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to upload to pypi'
+        required: true
+      pypi_repo:
+        description: 'Repo to upload to ("testpypi" or "pypi")'
+        default: 'testpypi'
+        required: true
+
+jobs:
+  publish:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - name: Install dependencies
+      run: |
+        python -m pip install -U pip
+        python -m pip install -U setuptools wheel twine build
+
+    - name: Generate distribution archives
+      run: |
+        python -m build
```

Comment:
oh, didn't know this does the thing.

Reply:
Same, it was new to me. But I figured if the [docs](https://packaging.python.org/en/latest/tutorials/packaging-projects/#generating-distribution-archives) recommend this way, it is best practice.